.github/workflows/benchstat.yml

```diff
@@ -0,0 +1,100 @@
+
+name: benchstat
+
+on:
+  - pull_request
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  benchstat:
+    strategy:
+      matrix:
+        go: ["1.21"]
```

Comment:
Can we read this from `.go-version` like we do elsewhere?

Reply:
I don't see a `.go-version` in a number of other libs. I can look to add one if that is useful, but I think that would make sense as a follow up PR.

Reply:
Ah, sorry, I thought this was the Boundary repo, carry on 😁.